### PR TITLE
Add redis as an option for database

### PR DESCRIPTION
**Pull request by schoolboybru**
By submitting this pull request, I confirm that my contribution is made under the terms of the MIT license.

## Problem/Feature

Adding Redis as an option for a database. 

## Description of Changes: 

1. Add all templates for redis
2. Make redis an option when selecting a database
3.  Update documentation 

## Checklist

- [x] I have self-reviewed the changes being requested
- [x] I have updated the documentation (if applicable)

### CODE REVIEW

File: cmd/template/dbdriver/files/service/redis.tmpl [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+	err := s.db.Ping(ctx)
+
+	if err != nil {
+		log.Fatalf(fmt.Sprintf("db down: %v", err))
+	}
+
+	return map[string]string{
+		"message": "It's healthy",
+	}
+}
```
Comment (Ujstor) [inline cmd/template/dbdriver/files/service/redis.tmpl:62]: Ping is returning an error, and it is breaking the app when you curl the health route. 

I think this solves the issue

```go
func (s *service) Health() map[string]string {
	ctx, cancel := context.WithTimeout(context.Background(), 1*time.Second)
	defer cancel()

	result, err := s.db.Ping(ctx).Result()

	if err != nil {
		log.Fatalf(fmt.Sprintf("db down: %v", err))
	}

	 if result != "PONG" {
        log.Fatalf("Unexpected ping response: %s", result)
    }

	return map[string]string{
		"message": "It's healthy",
	}
}
```

**Review by Ujstor — CHANGES_REQUESTED** [review]
After fixing the Health func, this is ready for merge

File: cmd/template/dbdriver/files/service/redis.tmpl (hunk 1 above)
Comment (schoolboybru) [reply cmd/template/dbdriver/files/service/redis.tmpl]: Oh good catch 👍

**Review by Ujstor — APPROVED** [review]
LGTM

**Pull request merged**